src/main/java/com/redhat/cloud/notifications/RestAction.java

```diff
@@ -55,7 +55,12 @@
     @JsonProperty("account_id")
     public String accountId;
 
-    public java.util.Map<String,Object> payload;
+    @NotNull
+    public List<RestEvent> events;
+
+    @NotNull
+    public String context;
```

Comment:
This maps 1-1 with what we have in the avro schema, but I think we can be less harsh with the user and use a `Map<String, Object>` instead (the same with the payload)